src/main/fc/fc_main.c

```diff
@@ -209,7 +209,7 @@ void processRcCommand(void)
     static int16_t deltaRC[4] = { 0, 0, 0, 0 };
     static int16_t factor, rcInterpolationFactor;
     static uint16_t currentRxRefreshRate;
-    const uint8_t interpolationChannels = rxConfig()->rcInterpolationChannels + 1;
+    const uint8_t interpolationChannels = rxConfig()->rcInterpolationChannels + YAW;
```

Comment:
The loop is `for(int channel=ROLL; channel < interpolationChannels; channel++) {`. It should be consistent (but both variants are fine for me)

Another possibility is to use channel bitmap (at least in fc_main) and convert user-specified value from CLI ...